pagerduty/data_source_pagerduty_licenses.go

```diff
@@ -0,0 +1,140 @@
+package pagerduty
+
+import (
+	"log"
+	"time"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/heimweh/go-pagerduty/pagerduty"
+)
+
+func dataSourcePagerDutyLicenses() *schema.Resource {
```

Comment:
Discussed with @imjaroiswebdev to create a singular data source and reference the schema in this file